README.md

```diff
@@ -12,6 +12,8 @@ The Cloud Foundation toolkit (henceforth, CFT) includes the following parts:
   single or multiple CFT-compliant config files - see:
   - The CFT source Python files (the `src/` directory)
   - The [CFT DM User Guide](dm/docs/userguide.md)
+- A [growing set of YAML deployments](config-connector/solutions/) for use with [Config
```

Comment:
To match the format above, should we say - A growing set of [Config Connector](link to KCC doc) examples: see config-connector/solutions ?

Reply:
+1

Reply:
Yes.

Reply:
Changed.